meta.txt

```diff
@@ -0,0 +1 @@
+010000000000010600000001070000000000000000010a0000001514480008287374616b696e675f696f2c496e69745374616b696e6700001001547374616b696e675f746f6b656e5f6164647265737304011c4163746f7249640001507265776172645f746f6b656e5f6164647265737304011c4163746f724964000144646973747269627574696f6e5f74696d6510010c7536340001307265776172645f746f74616c1401107531323800000410106773746418636f6d6d6f6e287072696d6974697665731c4163746f724964000004000801205b75383b2033325d000008000003200000000c000c00000503001000000506001400000507001808287374616b696e675f696f345374616b696e67416374696f6e000110145374616b65040014011075313238000000205769746864726177040014011075313238000100345570646174655374616b696e67040000012c496e69745374616b696e6700020024476574526577617264000300001c0418526573756c740804540120044501240108084f6b040020000000000c45727204002400000100002008287374616b696e675f696f305374616b696e674576656e74000110345374616b6541636365707465640400140110753132380000002457697468647261776e0400140110753132380001001c5570646174656400020018526577617264040014011075313238000300002408287374616b696e675f696f144572726f72000120284e756c6c416d6f756e74000000284e756c6c526577617264000100204e756c6c54696d65000200385472616e73666572546f6b656e730003006450726576696f757354784d7573744265436f6d706c6574656400040048496e737566666963656e7442616c616e6365000500204e6f744f776e6572000600385374616b65724e6f74466f756e64000700002808287374616b696e675f696f24496f5374616b696e6700003401146f776e657204011c4163746f7249640001547374616b696e675f746f6b656e5f6164647265737304011c4163746f7249640001507265776172645f746f6b656e5f6164647265737304011c4163746f724964000140746f6b656e735f7065725f7374616b6514011075313238000130746f74616c5f7374616b656414011075313238000144646973747269627574696f6e5f74696d6510010c75363400013470726f64756365645f74696d6510010c7536340001307265776172645f746f74616c14011075313238000130616c6c5f70726f64756365641401107531323800013c7265776172645f70726f64756365641401107531323800011c7374616b6572732c01585665633c284163746f7249642c205374616b6572293e0001307472616e73616374696f6e733801b442547265654d61703c4163746f7249642c205472616e73616374696f6e3c5374616b696e67416374696f6e3e3e00012c63757272656e745f7469641001345472616e73616374696f6e496400002c000002300030000004080434003408287374616b696e675f696f185374616b6572000010011c62616c616e6365140110753132380001387265776172645f616c6c6f7765641401107531323800012c7265776172645f646562741401107531323800012c646973747269627574656414011075313238000038042042547265654d617008044b01040456013c000400400000003c08287374616b696e675f696f2c5472616e73616374696f6e04045401180008010869641001345472616e73616374696f6e4964000118616374696f6e1801045400004000000244004400000408043c00
```

Comment:
Please sync with the template, add this file to `.gitignore`, and `git rm meta.txt`.